README.md

```diff
@@ -44,3 +44,25 @@ by communicating with the Linux kernel through `/dev/spidev*`. This means:
 
 If you modify `spi.pyx`, make sure to set the `USE_CYTHON` environment variable before building---otherwise your
 changes will not be compiled into `spi.c`.
+
+#### Running the code on Linux desktop
+
+Note that the cython code will not run on Windows.    
```

Comment:
```suggestion
Note that the python code will not run on Windows.    
```

Reply:
again, I'll add more clarification; the python code works on Windows, the Cython code uses some calls into the Linux system thus you cannot compile it.

Reply:
OK sorry, I thought it was a typo! I've heard of CPython but never cythong 😅